test/unit/controllers/CaseControllerSpec.scala

```diff
@@ -73,6 +73,7 @@ class CaseControllerSpec extends ControllerBaseSpec with BeforeAndAfterEach {
     atarController,
     correspondenceController,
     miscellaneousController,
+    redirectController,
```

Comment:
small change could we name this redirectService and in other controllerspecs

Reply:
nice catch, done!